pkg/kube/controllers/statefulset/statefulset_rollout_controller.go

```diff
@@ -0,0 +1,68 @@
+package statefulset
```

Comment:
This should be in the `quarksstatefulset` package

Reply:
@viovanov Actually the controller is usable standalone.

Reply:
What would be the benefit of having it in `quarksstatefulset`? I would also prefer to keep it independent if possible.